README.md

```diff
@@ -149,4 +149,4 @@ Vodka is pre-alpha and, at the time you are reading this, many things are probab
 
 ## License
 
-All code is open source licensed under the Greater GPLv3.
\ No newline at end of file
+All code is open source licensed under the Greater GPLv3.
```

Comment:
This is weird, maybe you added a newline to the end of the file? This is my first time reviewing code with this tool, sorry, still getting used to it